regression/failed-tests-printer.pl

```diff
@@ -6,20 +6,22 @@
 
 my $printed_this_test = 1;
 my $current_test = "";
+my $output_file = "";
 
 while (<LOG>) {
   chomp;
   if (/^Test '(.+)'/) {
     $current_test = $1;
     $printed_this_test = 0;
+  } elsif (/Output:\s+([^\s]+)/) {
+    $output_file = $1;
   } elsif (/\[FAILED\]\s*$/) {
     if(0 == $printed_this_test) {
+      print "'$current_test/$output_file'";
```

Comment:
Is this debug output? Note that the two blank lines printed next are meant to separate tests, so it may be better to place this output line further below (if it is meant to stay).